ImageUpdate/update_generic.sh

```diff
@@ -81,13 +87,16 @@ function configure_crony(){
     if [ "$_OS_DISTRO" == "ubuntu" ] || [ "$_OS_DISTRO" == "debian" ]; then
         CHRONY_FILE='/etc/chrony/chrony.conf'
     fi
-    virt-customize -a "${IMAGE_RELEASE}.img" --run-command "sed -i '1s/^/pool ntp.ugent.be iburst\n/' ${CHRONY_FILE}" --selinux-relabel
-    virt-customize -a "${IMAGE_RELEASE}.img" --run-command 'ln -sfn /usr/share/zoneinfo/Europe/Brussels /etc/localtime' --selinux-relabel
+    virt-customize -a "${TMP_DIR}/${IMAGE_RELEASE}.img" --run-command "sed -i '1s/^/pool ntp.ugent.be iburst\n/' ${CHRONY_FILE}" --selinux-relabel
+    virt-customize -a "${TMP_DIR}/${IMAGE_RELEASE}.img" --run-command 'ln -sfn /usr/share/zoneinfo/Europe/Brussels /etc/localtime' --selinux-relabel
     success "configured chrony"
 }
 function download_iso(){
-    wget "${URL}" -O "${IMAGE_RELEASE}.img"
-    success "Downloaded ${IMAGE_RELEASE}.img"
+    wget "${URL}" -O "${TMP_DIR}/${IMAGE_RELEASE}.img"
```

Comment:
if you run this in parallel you should check you still have room in the TMP_DIR, just in case, maybe  checking this with ```--spider``` may help